DeAceroDLabs/marketplace-frontend
# [DMP-495] Stepper component for cart page

## What is done

[[DMP-495] Cart Stepper](https://deacero.atlassian.net/browse/DMP-495)

## How is done

1. Modified the stepper component.
2. Created a new IconStep component.
3. Added a new route /carrito/:cartStep to verify if user is on cart or payment.

## How to test

1. Run `npm start`.
2. Login to the app.
3. Click on the shopping cart item.
4. You should be redirected to `/carrito` and see the new stepper.
5. click on the icons to change the route and colors.
